### PR TITLE
feat: Add pagination and more button at the bottom

### PR DESCRIPTION
**Pull request by gfortaine**
Fixes https://github.com/vercel/next-react-server-components/issues/56

### CODE REVIEW

File: app/head.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default async function Head() {
       />
       <meta charSet="utf-8" />
       <link href="/favicon.ico" rel="shortcut icon" />
-      <meta content="#ffa52a" name="theme-color" />
+      <meta content="#ff6600" name="theme-color" />
```
Comment (leerob) [inline app/head.js:13]: Let's keep the original color, unless there's a specific reason here?

**Comment by leerob** [comment]
If you navigate backward from an individual item, you get a 404.

CleanShot 2022-11-13 at 18 56 50@2x

**Comment by gfortaine** [comment]
@leerob Could you try again with the latest deployment link, please ?

https://next-rsc-news-4jbhmhha6.vercel.sh

Indeed, it looks like that the [link]() that you are mentioning doesn't work anymore.

Note: I will revert the changes regarding the links in the header to put them in another PR (it looks like disgraceful on mobile at the current time).

**Comment by gfortaine** [comment]
It seems OK on my side. However, it appears that I noticed a very interesting bug if you navigate backward from a more page (let's say `?p=5` => `?p=4`, by example). Given that Next.js do a soft navigation by default, we see the wrong results (the query parameter is not taken into account) : https://github.com/vercel/next.js/issues/42546#issuecomment-1305406746

**Comment by gfortaine** [comment]
It seems that `fetch` is not yet supported in Client Components. So, as suggested in the documentation, I used (SWR)[https://swr.vercel.app/] as a temporary workaround :

https://beta.nextjs.org/docs/data-fetching/fetching#use-in-client-components

**Comment by gfortaine** [comment]
> If you navigate backward from an individual item, you get a 404.
> 
> 
> 
> CleanShot 2022-11-13 at 18 56 50@2x
> 
> 

It looks like that this bug is happening during deployment. Indeed, it seems that I don't manage to reproduce it during local development.

**Comment by gfortaine** [comment]
> If you navigate backward from an individual item, you get a 404.
> 
> CleanShot 2022-11-13 at 18 56 50@2x

It should be fixed :+1:

**Comment by gfortaine** [comment]
> If you navigate backward from an individual item, you get a 404.
> 
> 
> 
> CleanShot 2022-11-13 at 18 56 50@2x
> 
> 

https://github.com/vercel/next.js/issues/40549

**Comment by leerob** [comment]
Thank you! 

I think there's still more things we want to do here, like make the transition for page to page more seamless, and have a better loading skeleton for the individual item page, but this is a great start. I really appreciate it.

**Pull request merged**